### DISCUSSION TITLE
Compatibility of GNU MCU Eclipse and Eclipse Embedded CDT

### DISCUSSION THREAD

**Discussion by mmx85** [eclipse-embed-cdt/eclipse-plugins#589] [Managed Build Plug-ins Q&A] [answered]
Hello.
I have a couple questions about the compatibility of GNU MCU Eclipse and Eclipse Embedded CDT plug-ins:
1. Are GNU MCU Eclipse 4.7.2 and Eclipse Embedded CDT 6.3.2 backward and forward compatible for the following use-case: Application is created on Eclipse with GNU MCU Eclipse plug-ins installed then updated on Eclipse with Eclipse Embedded CDT and then again opened on  GNU MCU Eclipse? This transition in general works for me but am I missing something?
2. When I create a new application on Eclipse with GNU MCU Eclipse plug-ins 4.7.2 installed and then open and save the project of Eclipse with Eclipse Embedded CDT plug-ins 6.3.2 installed then .cproject file is slightly modified.  For options like ilg.gnuarmeclipse.managedbuild.cross..... new argument "name" appears. Does it affects compatibility with Eclipse with GNU MCU Eclipse plug-ins 4.7.2 installed in any way? It looks like this: name="Display all headers (--all-headers|-x)". 
3. What does the value of "name" argument mean? Is it some command line parameter?

**Comment by ilg-ul** [comment, marked as answer]
Backward compatibility is not guaranteed and mixing incompatible versions may damage your projects.

**Reply by mmx85** [reply]
Thank you for reply.
